modules/ROOT/pages/boot-camp-tasks/software/setup.adoc

```diff
@@ -3,7 +3,8 @@
 . Install Linux to a provided computer.
 Use the provided USB stick.
 You should choose the simple install and overwrite any existing OS-es.
-.. Use `robot` for both the username and password.
+Use `robot` for both the username and password. You can use another version of Linux but consider, that a library we use `pyrealsense2` is only available on python 3.11 or older. The guide below applies for Ubuntu 22 and python 3.10.
```

Comment:
Write each sentence (or part of the sentence for long sentences) as a separate line for better diffs.